src/config.c

```diff
@@ -3095,6 +3095,7 @@ standardConfig static_configs[] = {
     createBoolConfig("latency-tracking", NULL, MODIFIABLE_CONFIG, server.latency_tracking_enabled, 1, NULL, NULL),
     createBoolConfig("aof-disable-auto-gc", NULL, MODIFIABLE_CONFIG | HIDDEN_CONFIG, server.aof_disable_auto_gc, 0, NULL, updateAofAutoGCEnabled),
     createBoolConfig("replica-ignore-disk-write-errors", NULL, MODIFIABLE_CONFIG, server.repl_ignore_disk_write_error, 0, NULL, NULL),
+    createBoolConfig("syslog-logfmt", NULL, MODIFIABLE_CONFIG, server.logfmt, 0, NULL, NULL),
```

Comment:
This is not specific to syslog, so syslog shouldn't be part of the name. By default Redis just logs to a file or to stdout, not to syslog.

I'm also thinking that maybe this should be an enum config so we can add more logging formats in the future. How about `log-format` with the options `default` and `logfmt`?

Reply:
I agree with enum!